### PR TITLE
feat: add shummg/textmodule

### LINKED ISSUE #251: 【パッケージの更新】textmodule
### 名前
しゅう

### データ
``` xml
<packages>
	<package>
		<id>shummg/textmodule</id>
		<name>textmodule</name>
		<overview>AviUtl用の様々な関数を提供する汎用モジュール</overview>
		<description>AviUtl用の様々な関数を提供するライブラリ群です。</description>
		<developer>しゅう</developer>
		<pageURL>https://shummg.work/archives/1157</pageURL>
		<downloadURL>https://github.com/shumm7/textmodule/releases/latest</downloadURL>
		<files>
			<file>script/textmodule.dll</file>
			<file>cpprest142_2_10.dll</file>
			<file directory="true">textmodule</file>
		</files>
	</package>
</packages>
```

### タイムスタンプ
2022/06/22 5:18:46

### PR DESCRIPTION
**Pull request by hal-shu-sato**
## Description

- Add `textmodule` by `shummg`.

## Related Issue

- close #251

## Checklist:

- [x] Updated the modification date in `mod.xml`.

### CODE REVIEW

File: v2/data/packages.xml [hunk 1]
```diff
@@ -3360,6 +3360,143 @@
...
+		<id>shummg/textmodule</id>
+		<name>textmodule</name>
+		<overview>AviUtl用の様々な関数を提供する汎用モジュール</overview>
+		<description>AviUtl用の様々な関数を提供するライブラリ群です。</description>
+		<developer>しゅう</developer>
+		<pageURL>https://shummg.work/archives/1157</pageURL>
+		<downloadURL>https://github.com/shumm7/textmodule/releases/latest</downloadURL>
+		<latestVersion>1.3.7</latestVersion>
+		<files>
+			<file>script/textmodule.dll</file>
```
Comment (mitosagi) [inline v2/data/packages.xml:3375]: As you noted, it only worked in the same folder as `exedit.auf`.

**Review by mitosagi — CHANGES_REQUESTED** [review]
I just want to confirm one thing.

File: v2/data/packages.xml (hunk 1 above)
Comment (hal-shu-sato) [reply v2/data/packages.xml:3375]: Oh...
I forget it.

**Review by mitosagi — DISMISSED** [review]
It looks good.

**Comment by hal-shu-sato** [comment]
Oh. I forgot converting.
So, I have pushed one more time.

**Review by mitosagi — APPROVED** [review]
Confirmed.

**Pull request merged**